### ISSUE TITLE
[Enhancement]: aws_docdbelastic_cluster: add argument 'backup-retention-period' and 'preferred-backup-window'

### ISSUE THREAD

**Issue by chuleui** [hashicorp/terraform-provider-aws#38940]
### Description

Currently, the aws_docdbelastic_cluster resource in Terraform does not support the backup-retention-period and preferred-backup-window arguments, which are critical for managing the backup configurations of Amazon DocumentDB (Elastic Clusters).

### Affected Resource(s) and/or Data Source(s)

aws_docdbelastic_cluster

### Potential Terraform Configuration

```terraform
resource "aws_docdbelastic_cluster" "example" {
  cluster_name           = "example"
  admin_user_name        = "admin"
  admin_user_password    = "password"
  backup_retention_period = 7
  preferred_backup_window = "03:00-04:00"
}
```

### Would you like to implement a fix?

None

**Comment by trutled3** [comment]
I am interested in working on this issue.

**Comment by johnsonaj** [comment]
This enhancement was released in `v5.66.0` with PR #38452.
